src/gen/objects.rs

```diff
@@ -68,6 +69,8 @@ pub fn generate_object(obj: &Object, type_helper: &dyn TypeHelperRenderer) -> da
                 return $(cls_name)._(ptr);
             }
 
+            // TODO: Bring back object lowering from main
```

Comment:
is this still accurate?

Reply:
Yeah, I noticed a missing lower function for objects... It may be in the backup, just needed to remind myself